models/todo.js

```diff
@@ -0,0 +1,19 @@
+module.exports = (sequelize, DataTypes) => {
+    var Sequelize = require('sequelize');
+    var sequelize = new Sequelize(process.env.DB_URL)
+    
+    return sequelize.define('todoentries', {
+        title: {
+            type: Sequelize.STRING,
+            unique: false
+        },
+        completed: {
+            type: Sequelize.BOOLEAN
+        }
+    },
+    {
+        createdAt: false,
+        updatedAt: false,
+        freezeTableName: true
```

Comment:
are you talking about `freezeTableName` or `createdAt`, `updatedAt`, AND `freezeTableName`?